service/tutorial/order/OrderServices.xml

```diff
@@ -1,27 +1,158 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
 
-    <!-- Sample service to get Order Header details by statusId using Entity Find -->
-    <service verb="get" noun="OrderHeadersByStatus">
+<!--1-->
+    <service verb="get" noun="FemalePersons">
         <description>
-            Service to get OrderHeader records based on the given statusId.
+            Records of all females.
+        </description>
+        <out-parameters>
+            <parameter name="orderList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.party.Person" list="orderList">
+                <econdition field-name="gender" value="F"/>
+            </entity-find>
+        </actions>
+    </service>
+<!--2-->
+    <service verb="get" noun="Persons">
+        <description>
+            Person with Married marital status.
+        </description>
+        <out-parameters>
+            <parameter name="orderList" type="List" />
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.party.Person" list="orderList">
+                <econdition field-name="maritalStatusEnumId" value="MarsMarried"/>
+            </entity-find>
+        </actions>
+    </service>
+<!--3-->
+    <service verb="get" noun="LatestOrders">
+        <description>
+            Latest order header records in descending order
+        </description>
+        <out-parameters>
+            <parameter name = "orderList" type="List"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="orders" list="orderList">
+                <order-by field-name = "placedDate"/>
```

Comment:
Will this return orders in ascending or descending order?

Reply:
It is ordered in ascending order w.r.t orderId.